astropy/stats/histogram.py

```diff
@@ -205,9 +205,14 @@ def freedman_bin_width(data, return_bins=False):
 
     v25, v75 = np.percentile(data, [25, 75])
     dx = 2 * (v75 - v25) / (n ** (1 / 3))
+    
+    def mad(array):
```

Comment:
Is a nested function really necessary here?

Reply:
Ah...since the function is quite simple, the nested function is not necessary...

Reply:
The function isn't needed at all -- median_absolute_deviation is already a function in the astropy.stats package

Reply:
@crawfordsm thanks! I found I mistook median absolute deviation as mean absolute deviation before. I have updated it with the median_absolute_deviation from astropy.stats package.